components/home/About/About.tsx

```diff
@@ -32,10 +32,11 @@ export default function About() {
             </div>
           </div>
         </Column>
-        <Column lg={6} sm={7} xsmOrder={1}>
+        <Column lg={1} />
```

Comment:
Daria para matar essa coluna quando for mobile?
```suggestion
        <Column lg={1} sm={0} />
```

Reply:
Sim, eu percebi isso e acabei corrigindo na PR #26 hahaha